rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -21,9 +21,27 @@
     cache_valid_time: 600
   with_items: maas_apt_packages
 
-- name: Install pip packages
+- name: Create venv dir for maas
+  file: path={{ maas_venv }} state=directory
+
+- name: Copy pip conf to venv
+  copy: src=/root/.pip/pip.conf dest="{{ maas_venv }}/pip.conf"
+
+- name: Update pip
+  pip:
+    name: pip
+    extra_args: '--upgrade'
+    virtualenv: "{{ maas_venv_enabled | bool | ternary(maas_venv_bin|dirname,omit) }}"
```

Comment:
Why not `ternary(mass_venv, omit)`? `maas_venv_bin` is essentially `{{ maas_venv }}/bin`... no?